storage/ndb/rest-server/rest-api-server/internal/integrationtests/utils.go

```diff
@@ -478,7 +478,7 @@ func sendGRPCBatchRequest(t *testing.T, testInfo api.BatchOperationTestInfo) (in
 
 	// Create gRPC client
 	conf := config.GetAll()
-	conn, err := testutils.CreateGrpcConn(t, conf.Security.UseHopsworksAPIKeys, conf.Security.RequireAndVerifyClientCert)
+	conn, err := testutils.CreateGrpcConn(t, conf.Security.UseHopsworksAPIKeys, conf.Security.EnableTLS)
```

Comment:
This removes the test case where TLS is enabled, but the client doesn't use it. The config validation stops you from setting `RequireAndVerifyClientCert=true` && `conf.Security.EnableTLS=false` if that's what you're worried about.

Reply:
CreateGrpcConn has the following signature 
```
func CreateGrpcConn(t testing.TB, withAuth, withTLS bool) (*grpc.ClientConn, error) {
```

It does not have a param for RequireAndVerifyClientCert. if you run the tests such that you set the RequireAndVerifyClientCert=false in the config file then all the grpc tests will fail without this fix

Reply:
Why do they fail? I don't think they should?

Reply:
Please see the above comment. you were passing RequireAndVerifyClientCert to EnableTLS param.

Reply:
Yeah, but from the client side. So if `RequireAndVerifyClientCert=true`, then `EnableTLS=true` has to be true on the server side.  If `RequireAndVerifyClientCert=false`, the server can be running both `EnableTLS=true | false`, so we can quickly test that as well. Or am I misunderstanding something about TLS?

Reply:
And so I'm just curious why it would fail?